src/cli/README.md

```diff
@@ -1861,23 +1861,30 @@ Set the Thread Leader Weight.
 Done
 ```
 
-### linkmetrics mgmt \<ipaddr\> enhanced-ack clear
+### linkmetrics mgmt \[sync\] \<ipaddr\> enhanced-ack clear
```

Comment:
Should the default be "sync" mode?

Reply:
The current default mode is `async`. We can change the default mode to `sync` mode. Then it changes the default action of the command. If users have used these commands in their script, the change will cause unexpected issues. Users have to spend time to find out root cause and resolve the issue. So I think it may be better to not change the default action of the command. Thoughts?

Reply:
I would really like to see the default behavior be consistent across all CLI commands.

Maybe we can change the CLI command to use a different string and throw an error when using the legacy string. For example, something other than `linkmetrics mgmt`.

Reply:
Great! I will change the code to set the default mode to `sync`.

Reply:
I have renamed commands `linkmetrics mgmt` and `linkmetrics query` to `linkmetrics config` and `linkmetrics request`, and set these commands to `sync` mode by default.